### ISSUE TITLE
White flash when pushing/popping a view controller

### ISSUE THREAD

**Issue by m-revetria** [CosmicMind/Motion#4]
I want to push/pop a view controller with just a fade-in fade-out transition between them. I couldn't achieved it, instead I see the screen is flashing white in the middle of the transition. Please check out the GIF provided below for clarification.

Is there a way to have just a fade transition when pushing/popping view controllers?

![motion_flash_issue](https://cloud.githubusercontent.com/assets/4791678/23070119/c7795790-f508-11e6-9230-30b3a3b2ccfc.gif)

**Comment by daniel-jonathan** [comment]
I will take a look at this.

**Comment by daniel-jonathan** [comment]
Can you show me your code setup? As well, do you have a background color on your view controllers that are being transitioned?

**Comment by m-revetria** [comment]
Hi, after reading your comment I tried to set `backgroundColor` of the view controllers to `.clear` and the white flash just disappeared but now there is no fade animation.

![motion_flash_issue](https://cloud.githubusercontent.com/assets/4791678/23129940/4b4cb0e2-f763-11e6-97b7-59f1253af600.gif)

I'm enabling Motion in `viewDidLoad` from both view controllers and also in the container navigation controller.

In the first view controller I have a UICollectionView and when I touch some cell I want to transition to the next view controller with a fade effect and moving the image view to its final destination in the pushed view controller. That's all the configuration that I'm using so far. So I've set up the image view's motionIdentifier in both view controllers to be the same

```swift
/******** In the first view controller: ********/
override func viewDidLoad() {
    super.viewDidLoad()
    isMotionEnabled = true
}

func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
    let cell: PSGameCollectionViewCell! = collectionView.cellForItem(at: indexPath) as? PSGameCollectionViewCell
    cell.imageView.motionIdentifier = "gameCover"
    let gameData = gamesData[indexPath.item]
    // This is a `show` segue
    performSegue(withIdentifier: R.segue.pSDashboardViewController.showGameDetail, sender: gameData)
}

/******** In the second (pushed) view controller: ********/
override func viewDidLoad() {
    super.viewDidLoad()
    isMotionEnabled = true
    animatableCoverImageView.motionIdentifier = "gameCover"
}
```

This is what I'm trying to reproduce using Motion: https://dribbble.com/shots/3116935-Playstation-Network-Mobile-APP

**Comment by daniel-jonathan** [comment]
Okay, I am going to make an addition that will help with making the fade work. I have some other additions I am making today to Motion, so I will update you through this issue once I am done. Thank you!

**Comment by daniel-jonathan** [comment]
This should be fixed and will be pushed to `master` in the next release. Thank you!

**Comment by daniel-jonathan** [comment]
The fix for this issue is in [Motion 1.1.0](https://github.com/CosmicMind/Motion/releases/tag/1.1.0).
